src/main/scala/mesosphere/marathon/core/group/impl/GroupManagerImpl.scala

```diff
@@ -120,7 +120,8 @@ class GroupManagerImpl(
             Left(left)
           case Right(changed) =>
             val unversioned = assignDynamicServicePorts(from, changed)
-            val to = GroupVersioningUtil.updateVersionInfoForChangedApps(version, from, unversioned)
+            val toApps = GroupVersioningUtil.updateVersionInfoForChangedApps(version, from, unversioned)
```

Comment:
nit: `toApps` and `to` are bad names. suggest: `withUpdatedApps` and `withUpdatedAppsPods`